front/src/applications/stdcmV2/components/StdcmConsist.tsx

```diff
@@ -129,22 +153,38 @@ const StdcmConsist = ({ setCurrentSimulationInputs, disabled = false }: StdcmCon
           id="tonnage"
           label={t('consist.tonnage')}
           trailingContent="t"
-          inputFieldWrapperClassname="weight"
+          type="number"
+          min={0}
```

Comment:
I can put negative number, we should do something about that probably. @thibautsailly how do you want to handle this ?

Reply:
Yes, the min props is not a validator, but it works when you use the up and down keys.